sw/tools/generators/gen_aircraft.ml

```diff
@@ -397,9 +397,9 @@ let () =
 
     (* Get TARGET env, needed to build modules.h according to the target *)
     let t = try Printf.sprintf "TARGET=%s" (Sys.getenv "TARGET") with _ -> "" in
+    make_opt "radio_ac_h" "RADIO" "radio";
```

Comment:
Wondering what's up with that change?
Did that result in wrong deps in the old order?